demisto_sdk/dev_tools/spell_checker.py

```diff
@@ -0,0 +1,95 @@
+import yaml
+
+from spellchecker import SpellChecker
+from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS
+from demisto_sdk.common.constants import DISPLAYABLE_LINES, SCRIPT_ARGS, KNOWN_WORDS
+
+
+class SpellCheck:
+    """
+        Attributes:
+            path (str): The path to the current file being checked
+    """
+
+    def __init__(self, path: str):
+        self.path = path
+        self.spellchecker = SpellChecker()
+
+    def run_spell_check(self):
+        """Runs spell-check on the given file.
+
+        Returns:
+            int. 0 if no problematic words found, 1 otherwise.
+        """
+        unknown_words = set([])
+        self.spellchecker.word_frequency.load_words(KNOWN_WORDS)
```

Comment:
This line simply making these words known to the spell checker, right? 
If so let's add an explaining comment to it

Reply:
done

Reply:
maybe this should be done in the init

Reply:
moved this to the attributes.